### PR TITLE
Cypress/E2E: Fix LDAP login and team/channel roles

### PR DESCRIPTION
**Pull request by saturninoabril**
#### Summary
Fix LDAP login and team/channel roles

#### Ticket Link
none, failing on daily tests

### CODE REVIEW

File: e2e/cypress/integration/enterprise/ldap/ldap_login_spec.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ context('ldap', () => {
             cy.apiAdminLogin().then(() => {
                 cy.apiUpdateConfig(ldapSetting).then(() => {
                     cy.doLDAPLogin(testSettings).then(() => {
-                        cy.get('#createPublicChannel').should('be.visible');
+                        cy.findAllByLabelText('town square public channel').should('be.visible');
```
Comment (josephbaylon) [inline e2e/cypress/integration/enterprise/ldap/ldap_login_spec.js:118]: @saturninoabril I'm getting this failure when I run the test locally

Screen Shot 2020-08-12 at 10 27 08 AM

Comment (saturninoabril) [reply e2e/cypress/integration/enterprise/ldap/ldap_login_spec.js]: Thanks for checking. Pass on me but not sure why it's failing on you. I added a URL check and timeout. Please try again and let me know how it works for you.

Comment (josephbaylon) [reply e2e/cypress/integration/enterprise/ldap/ldap_login_spec.js]: That fixed it

**Comment by prapti** [comment]
@saturninoabril The initial issue that @josephbaylon pointed out is no longer there for me, but seeing some other issues now. The following error always happens when I run the login flow for guest test. Not sure if it's my setup that's missing something if the tests are passing for you and Joseph:
Screen Shot 2020-08-12 at 12 11 20 PM

**Pull request merged**